space-plugins/nextjs-starter/README.md

```diff
@@ -129,3 +129,23 @@ For more detailed information on Storyblok extensions, read the following guides
 
 - [Space Plugin](https://www.storyblok.com/docs/plugins/custom-application)
 - [OAuth 2.0 Authorization Flow](https://www.storyblok.com/docs/plugins/authentication-apps)
+
+
+## Troubleshooting
```

Comment:
Once this is reviewed, I will replicate it for 

- space-plugin/nuxt-starter
- tool-plugin/nextjs-starter
- tool-plugin/nuxt-starter

Reply:
Btw, anything else that we should mention here?

Reply:
Shall we include a couple of screenshots? At least for the Extensions Settings part will be nice 🫂

Reply:
here it is! :)
https://github.com/storyblok/space-tool-plugins/blob/bc7c550e31a3d22025bc516ec90df0d1d5a507d9/space-plugins/nextjs-starter/README.md